examples/example-gateway/config/production.yaml

```diff
@@ -67,6 +67,17 @@ tchannel.processName: example-gateway
 tchannel.serviceName: example-gateway
 tchannel.clients.requestUUIDHeaderKey: x-request-uuid
 useDatacenter: false
-
+clients.baz.alternates: {"routingConfigs":[{"headerName":"x-api-environment","headerValue":"\\*",
```

Comment:
Can you include YAML annotations on the struct as well so this can be defined as YAML?